comm/src/test/scala/coop/rchain/kademlia/distance.scala

```diff
@@ -36,109 +36,113 @@ class DistanceSpec extends FlatSpec with Matchers {
     }
   }
 
-  val width = 32
-
-  // Make 8*width copies all of which differ in a single, distinct bit
-  def oneOffs(key: Seq[Byte]): Seq[Array[Byte]] =
-    for {
-      i <- 0 until width
-      j <- 7 to 0 by -1
-    } yield {
-      val k1 = Array.fill(key.size)(b(0))
-      Array.copy(k1, 0, key.toArray, 0, key.size)
-      k1(i) = b(k1(i) ^ b(1 << j))
-      k1
+  for (exp <- 1 to 8) {
+
+    // toDouble gets rid of a compiler warning about numeric widening
+    val width = scala.math.pow(2, exp.toDouble).toInt
```

Comment:
Does `1 << exp` give the warning?

Reply:
Haha. Why didn't I just do that?! The rest of the code is littered with it.